notebooks/NIRCam/NIRCam_claw_detection/nircam_claw_detection.ipynb

```diff
@@ -0,0 +1,461 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3e916d4f-ffe0-4c07-8c7c-a122969bda36",
+   "metadata": {},
+   "source": [
+    "# NIRCam Claw Detection.\n",
+    "---\n",
+    "**Author**: Mees Fix (mfix@stsci.edu) <br>\n",
+    "**Latest Update**: 05 March 2025<br>\n",
+    "**Use case**: NIRCam Imaging detectors A1, A3, B1, B2 or B4. <br>\n",
+    "**Data**: None<br>\n",
+    "**Test Pipeline Version**: None"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee829f57-d20d-4989-aea5-b6114dbd116d",
+   "metadata": {},
+   "source": [
+    "<a id='intro'></a>\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook demonstrates how to use JWST pointing information provided in a user's APT program to detect the potential of\n",
+    "[claws](https://jwst-docs.stsci.edu/known-issues-with-jwst-data/nircam-known-issues/nircam-scattered-light-artifacts#NIRCamScatteredLightArtifacts-clawsClaws) in NIRCam observations. Claws are a scattered light phenomenon that occur when a bright star falls in the susceptibility region. When present, claws occur primarily in the A1, A3, B1, B2 or B4 detectors, affecting roughly 5% of the pixels on those detectors.\n",
+    "\n",
+    "All datasets necessary for this notebook to run are included in the [jwst_rogue_path_tool](https://github.com/spacetelescope/jwst_rogue_path_tool) python package. This includes the following:\n",
+    "\n",
+    "* `APT_test_4RPtool.records.json`:  A mock program that contains NIRCam instrument configurations that could contain contain claws we want to detect.\n",
+    "* `rogue_path_nrca.fits`: fits file that contains the susceptibility region intensity contours for NIRCam module A. \n",
+    "* `rogue_path_nrcb.fits`: fits file that contains the susceptibility region intensity contours for NIRCam module B.\n",
+    "* `two_mass_kmag_lt_5.csv`: [2MASS](https://irsa.ipac.caltech.edu/Missions/2mass.html) stellar sources that have magnitudes less than 5.\n",
+    "\n",
+    "To generate your own `records.json` file for a specific program via APT, see the [JDox page for APT Export files](https://jwst-docs.stsci.edu/jwst-astronomers-proposal-tool-overview/additional-jwst-apt-functionality/apt-export-files#APTExportFiles-records_filerecordsfile&gsc.tab=0)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "314dcac9-fb10-4c0e-8fc2-d49bd17415d3",
+   "metadata": {},
+   "source": [
+    "<a id='top'></a>\n",
+    "## Table of Contents {-}\n",
+    "* [Imports](#imports)\n",
+    "* [Run the JWST Rogue Path Tool](#Run-jwst-rpt)\n",
+    "    * [Handling records.json Files](#handle-records-json-files)\n",
+    "    * [Selecting and Displaying APT Data](#Selecting-displaying-program-data)\n",
+    "    * [Generating APT Program Object](Generating-apt-program-object)\n",
```

Comment:
missing `#` here. Also, I'm not sure why, but clicking on any of these table of contents links doesn't take me to the section.

Reply:
Changing to just using the section name worked though, e.g. 

`* [Reporting & Applying PA Constraints in APT](#Reporting-&-Applying-PA-Constraints-in-APT)`